src/mlpack/bindings/python/setup.py.in

```diff
@@ -34,8 +34,7 @@ else:
 # directories with a (valid) space in the name will be given to us as '\ '; so,
 # in order to split these right, we first convert all spaces to ';', then
 # convert '\;' back to ' ', then split on ';'.
-library_dirs = list(filter(None, ['${MLPACK_LIBDIR}'] +
-    '${Boost_LIBRARY_DIRS}'.replace(' ', ';').replace('\;', ' ').split(' ')))
+library_dirs = list(filter(None, ['${MLPACK_LIBDIR}']))
```

Comment:
```suggestion
library_dirs = ['${MLPACK_LIBDIR}']
```

I think that this would work too?  I'm not sure why there was a `filter(None, ...)` to begin with.  It seems like that wouldn't do anything.